src/DataModel/Configuration/ItemCrafting/SimpleCraftingSettings.cs

```diff
@@ -22,11 +22,22 @@ public partial class SimpleCraftingSettings
     /// </summary>
     public int MoneyPerFinalSuccessPercentage { get; set; }
 
+    /// <summary>
+    /// Gets or sets the NPC price divisor for the sum of crafting items' prices. For each full division, the percentage gets increased by 1 percent, and the mix price rises.
+    /// </summary>
+    /// <remarks>Used for Chaos Weapon and 1st Level Wing craftings.</remarks>
+    public int NpcPriceDivisor { get; set; }
```

Comment:
The individual `NpcPriceDivisor` per item was not enough for Chaos Weapon and 1st Level Wing craftings, as it's the sum value of all the items (even jewels) that is divided.
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L291